src/components/SchemaTable/index.jsx

```diff
@@ -65,23 +132,9 @@ function SchemaTable({ schema }) {
    * These will then be stored into 'leftRows' and 'rightRows' arrays each.
    */
   const renderDefault = schemaInput => {
-    const leftRow = (
-      <NormalLeftRow
-        key={leftRows.length + 1}
-        schema={schemaInput}
-        classes={classes}
-      />
-    );
-    const rightRow = (
-      <NormalRightRow
-        key={rightRows.length + 1}
-        schema={schemaInput}
-        classes={classes}
-      />
-    );
+    const normalRow = createNormalRow(schemaInput);
 
-    leftRows.push(leftRow);
-    rightRows.push(rightRow);
+    pushRow(normalRow);
   };
 
   /**
```

Comment:
![Screen Shot 2019-12-16 at 1 34 07 PM](https://user-images.githubusercontent.com/3766511/70932935-da266d00-2008-11ea-8109-1268131cff78.png)

I feel like the border is doubled there.

Reply:
Updated in commit 1dc70ba !